config.go

```diff
@@ -116,7 +116,7 @@ func (c *DatabaseConfig) Validate() error {
 	return nil
 }
 
-func (c *DatabaseConfig) SqlDB(logger *logrus.Entry) (*sql.DB, error) {
+func (c *DatabaseConfig) SqlDB(logger *logrus.Entry, comment string) (*sql.DB, error) {
```

Comment:
Can this comment not be a struct member on `DatabaseConfig`? It feels like a configuration to me. If there's a concern of needing to specify this for the source master, source, and target databases, we can have `Config.QueryComment` be automatically propagated to each `DatabaseConfig` near the code where we set defaults?